bin/spark-stats.js

```diff
@@ -0,0 +1,75 @@
+import http from 'node:http'
+import { once } from 'node:events'
+import pg from 'pg'
+import Sentry from '@sentry/node'
+import fs from 'node:fs/promises'
+import { join, dirname } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import { createHandler } from '../lib/handler.js'
+import { DATABASE_URL } from '../lib/config.js'
+
+const {
+  PORT = 8080,
+  HOST = '127.0.0.1',
+  SENTRY_ENVIRONMENT = 'development',
+  REQUEST_LOGGING = 'true'
+} = process.env
+
+const pkg = JSON.parse(
+  await fs.readFile(
+    join(
+      dirname(fileURLToPath(import.meta.url)),
+      '..',
+      'package.json'
+    ),
+    'utf8'
+  )
+)
+
+Sentry.init({
+  dsn: 'https://47b65848a6171ecd8bf9f5395a782b3f@o1408530.ingest.sentry.io/4506576125427712',
+  release: pkg.version,
+  environment: SENTRY_ENVIRONMENT,
+  tracesSampleRate: 0.1
+})
+
+const pgPool = new pg.Pool({
+  connectionString: DATABASE_URL,
+  // allow the pool to close all connections and become empty
+  min: 0,
+  // this values should correlate with service concurrency hard_limit configured in fly.toml
```

Comment:
can you add a matching comment to fly.toml, so that we don't forget updating here if that changes?

Reply:
Great idea 👍🏻 

- https://github.com/filecoin-station/spark-api/pull/194
- https://github.com/filecoin-station/spark-stats/pull/3